app/src/routes.js

```diff
@@ -270,7 +270,7 @@ export default (store) => {
       component: require("account/RevealSeedModal").default,
     },
     {
-      path: `/delete/:networkId/:address`,
+      path: `/delete/:address/:addressNetworkId`,
```

Comment:
why the change?

Reply:
networkId was being problematic in the `setNetwork` function since it is the same name for the network param. The order doesn't actually matter. I can restore the previous order

Reply:
alright, and why the reordering?

Reply:
leave it, but always better to minimize changes to not create confusion